### ISSUE TITLE
Craft 3 Image doesn't change after save when editing from matrix

### ISSUE THREAD

**Issue by michalbulla** [craftcms/cms#4217]
### Description
After editing an image in the matrix block and saving the asset, the changes doesn't affect the image thumbnail preview. The thumbnail preview changes only after saving the page.

### Steps to reproduce

1. Create matrix field with assets field type, assign this field to a section.
2. Create a new entry in that section, add an Image into the Assets Field. 
3. Double-click on the Image and edit it(crop for example). 
4. Save the Asset
4. The Thumbnail preview doesn't change. 

### Additional info

- Craft version: 3.1.24
- PHP version: 7.0.24
- Database driver & version: MySQL 5.6.15
- Plugins & versions: Architect 2.3.1, Redactor 2.3.2, Table Maker 2.0.1, Tungsten 2.0.2

**Comment by benface** [comment]
This is a duplicate of #4212
